### ISSUE TITLE
Dynamic Generated Page Still Need a Fix

### ISSUE THREAD

**Issue by taufik-nurrohman** [morfy-cms/morfy#9]
How could I fix this? The page appears fine in the browser but shows as error 404 in the statistics. If the sitemap plugin also displaying this, is this mean that they cannot be indexed by search engines?

![2014-02-12_210615](https://f.cloud.github.com/assets/1669261/2152204/800e1a54-9416-11e3-82f8-98666a5b60b7.png)
![2014-02-13_014703](https://f.cloud.github.com/assets/1669261/2152206/875fe0b2-9416-11e3-9fc4-c007dc1c48bd.png)

Basically, the generated pages that created via `before_render` are just a canceled **404** page.

https://github.com/Awilum/morfy-cms/blob/master/libraries/Morfy/Morfy.php#L390
